parkbuhyoen/19.6.py

```diff
@@ -0,0 +1,12 @@
+height=int(input())
+
```

Comment:
```suggestion
for i in range(1, a+1):
    print(' '*(a-i), '*'*(i*2-1), sep="", end="\n")
```
이런 방법으로 출력하는 것도 괜찮을 것 같아요!